libqfieldsync/project_checker.py

```diff
@@ -410,3 +415,22 @@ def check_layer_package_prevention(
             return FeedbackResult(main_msg)
 
         return None
+
+    def check_external_layers(
+        self, layer_source: LayerSource
+    ) -> Optional[FeedbackResult]:
+        home_path = Path(self.project.fileName()).parent
+
+        if layer_source.is_file and not layer_source.is_localized_path:
+            layer_path = Path(layer_source.layer.source()).resolve()
+
+            if home_path not in layer_path.parents:
+                return FeedbackResult(
+                    self.tr(
+                        "Layer '{}' is outside the project's home directory."
+                        "This layers may cause issues."
+                        "Please move the layer within '{}'."
```

Comment:
```suggestion
                        'Layer "{}" is outside the project's home directory.'
                        "This layers may cause issues."
                        'Please move the layer within "{}".'
```

I think pretty much all the code shows double quotes to the end users.

Reply:
Great, really thanks!